kotlinx-coroutines-core/build.gradle

```diff
@@ -118,6 +111,25 @@ configurations {
     configureKotlinJvmPlatform(kotlinCompilerPluginClasspath)
 }
 
+if (project.ext.ideaActive) {
+    def ideaPreset = kotlin.presets.linuxX64
+    if (project.ext.isLinuxHost) ideaPreset = kotlin.presets.macosX64
```

Comment:
Is this intended?
if **isLinuxHost** then ideaPreset = **macos**

Reply:
Good catch. Thanks.